src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -139,8 +125,8 @@ private static SymbolLookup libLookup(Function<RawNativeLibraries, NativeLibrary
      * Returns the path of the given library name from JDK
      */
     private static Path jdkLibraryPath(String name) {
-        Path javahome = Path.of(GetPropertyAction.privilegedGetProperty("java.home"));
-        String lib = Utils.IS_WINDOWS ? "bin" : "lib";
+        Path javahome = Path.of(System.getProperty("java.home"));
```

Comment:
If not out of scope for this PR:
System property java.home is available in a static; will need an `import jdk.internal.util.StaticProperty`
```suggestion
        Path javahome = StaticProperty.javaHome();
```

Reply:
Let's do that change under a separate PR.

Reply:
https://bugs.openjdk.org/browse/JDK-8344134